Antistasi-Ultimate-Community/A3-Antistasi-Ultimate
# Unit tier changes

## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Changed the war level at which units spawn:
militia 1-3;
military 4-7;
elite 8+
Also added CBA settings so the player can customise it themselves, even mid-game

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps:
Load any template, see if everything still works (outposts still spawn all units at higher war levels, etc)

********************************************************
Notes:
